.github/workflows/ci.yaml

```diff
@@ -23,6 +23,7 @@ jobs:
           sudo apt-get -y --no-install-recommends install wget
           docker pull $TR_IMAGE
           wget http://vanga.sista.arizona.edu/automates_data/vectors.txt -O $GITHUB_WORKSPACE/automates/text_reading/src/test/resources/vectors.txt
+          cp $GITHUB_WORKSPACE/automates/text_reading/src/test/resources/vectors.txt $GITHUB_WORKSPACE/automates/text_reading/vectors.txt
```

Comment:
@maxaalexeeva the `vectors.txt` file is only used for running `sbt test`. If you only need it to be located at `$GITHUB_WORKSPACE/automates/text_reading/vectors.txt` for the tests to pass then lets locate it there and not have it in two places. However, if it is needed in both locations then we can leave the file as it is currently.

Reply:
I now also need vectors some place else in main, so things break down if they are not in both places (main and test) :(

Reply:
To follow up on this conversation, if you need to use this under tests and in the actual application, you should be able to have it just in the actual application and then it should be available to the tests too (At least that's how it works for java apps and I would assume its the same for Scala).

Reply:
Ah I will try that. I think I misunderstood what Paul said. Makes sense. I'll try to fix that today.

Reply:
@dpdicken fixed